scheduleit/static/js/show.js

```diff
@@ -4,22 +4,22 @@ $(document).ready(function () {
   var url = $(this).parent().parent().children().eq(4).text();
   var slotHash =  $(this).parent().parent().children().eq(6).text();
   var onid =  $(this).parent().parent().children().eq(5).text();
-
+  var eventName = $(this).parent().parent().children().eq(7).text();
```

Comment:
This is a fragile way to get the event name and is based on the layout never changing. I thought you previously had these as data attributes on the button?